scripts/control-status

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+usage="
+$0: Get rough list of implementation statuses from control markdown files
+
+Usage:
+  $0 -h
+  $0 [-s INCLUDE_STATUS_LIST] [-i IGNORED_STATUS_LIST] [-m MARKDOWN_DIR]
+
+Options:
+-h: show help and exit
+-s: status names to include, given as comma separated list. Defaults to all controls
+-i: status names to ignore, given as comma separated list.
+-m: Directory containing markdown files. Defaults to 'markdown' value in config, or 'control-statements'
+
+Notes:
+* passing '-s' will take precedence over '-i'
+"
+
+set -e
+
+source /app/bin/functions.sh
+markdown=$(yaml_parse_value 'trestle-config.yaml' 'markdown' 'control-statements')
+ignored=""
+status=""
+
+while getopts "hs:i:m:" opt; do
+  case "$opt" in
+    s)
+      status=`sed s/,/"\\\\\|"/g <<< ${OPTARG}`
+      ;;
+    i)
+      ignored=`sed s/,/"\\\\\|"/g <<< ${OPTARG}`
+      ;;
+    m)
+      markdown=${OPTARG}
+      ;;
+    h)
+      echo "$usage"
+      exit 0
+      ;;
+  esac
+done
+
+echo "==========================================================================="
+echo "Gap report for /app/docs/$markdown"
+echo "==========================================================================="
+
+result=`grep -R "Implementation Status:" /app/docs/"$markdown"/* | cut -d':' -f1,3 | cut -d'/' -f5`
```

Comment:
question: We used `status:` as front matter in the Markdown in the past. Do we have a sample with the `Implementation Status:` to test this against?

Reply:
Try https://github.com/GSA-TTS/continuous-assessment-poc/blob/main/doc/compliance/oscal/control-statements/sc-28.1.md and the other files in that directory

Reply:
`Implementation Status: (partial|planned|implemented|etc...)` is a trestle-native part of the markdown and gets nicely transferred into the SSP so I prefer it to a custom front matter

Reply:
Ah ha! Yes, that works!